docs/cloud/integrations/issues.md

```diff
@@ -0,0 +1,25 @@
+# Issues
+
+Issues represent possibly problematic states detected within the Event Store Cloud.
+
+## Examples
+
+Note: these examples are a subset of issues created by the system. The exact details of why issues are created are subject to change, but the cause of the issue and steps to resolve it will hopefully be clear in the messages associated with the related events.
+
+### Core Load Count
+
+For each node of a cluster, the core load average is measured and divided by the number of physical cores on the node. If the result exceeds 2.0 an issue is opened. This issue is closed when the result consistently dips under 2.0.
+
+If this happens consider increasing the size of the instance type for the cluster.
```

Comment:
**COMMENT:** I understand that this is a shorthand, but this may be a bit misleading as we don't have, right now, an option to increase the size of the instance. We need to create a new cluster and restore a backup. I added a separate issue to document instructions on that. Then it could be linked here.